pwndbg/__init__.py

```diff
@@ -72,6 +73,12 @@
 except:
     pass
 
+# this is an unconventional workaround to
+# support unicode printing for python2
+# https://github.com/pwndbg/pwndbg/issues/117
+if six.PY2:
+    reload(sys)
+    sys.setdefaultencoding('utf-8')
```

Comment:
I think this will cause more issues than it solves when trying to render colors, no?